tests/nrf5340_audio/fakes/lc3_file/lc3_file_fake_data.c

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "lc3_file_fake_data.h"
+
+#include <zephyr/sys/util.h>
+
+const uint8_t fake_dataset1_valid[] = {0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
```

Comment:
The test you're thinking of was merged with the [lc3_file module](https://github.com/nrfconnect/sdk-nrf/commit/39cea52d39ab8e9c6f5ec089954d2e9aa2e7aa13).

My intention has been to write this module to not be closely tied to the LC3 format, with the idea that the lc3_file module will handle all this. This is also the reason I've made the tests as simple as they are for this module, since the responsibility for verifying the file is on the other module.

This might be a bit much to do in comments, but I'm happy to discuss it further offline if you want to 👍